FitOn/settings.py

```diff
@@ -1,19 +1,68 @@
 import os
 from pathlib import Path
 from datetime import timedelta
+import boto3
+import os
 
 # Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
 
 
+
+def get_secrets():
+    client = boto3.client('secretsmanager')
+    response = client.get_secret_value(
+        SecretId='googleFit_credentials'
+    )
+    print(response)
+    return response['GOOGLEFIT_CLIENT_ID'],response['GOOGLEFIT_CLIENT_SECRET']
+
+
+
+
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
 SECRET_KEY = 'django-insecure-iqw@@a4osoerv=_))5ipw&kthcyr@v55xwz#=sse!13()+s#l_'
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = False
+
+ALLOWED_HOSTS = ['*']
```

Comment:
ALLOWED_HOSTS is already present in line number 67 below

Reply:
Resolved